Updated Simplified Chinese translation files based on 1.16-materials-update

VeroFess opened the pull request:
Translated based on the latest branch; corrected some previous translation errors.

VeroFess committed:
Updated Simplified Chinese translation files based on 1.16-materials-update

Translated based on the latest branch; corrected some previous translation errors.

KnightMiner commented:
Alright, I have the last of the lang changes done for that branch. Since the version you did in your PR, new things include:
* Liquid Soul, Soul Glass, Soul Glass Pane (named after soul sand, so translate similarly)
* All the new materials have names (first new one is at `material.tconstruct.seared_stone`)

Add those two things and I think this should be good to merge

VeroFess committed:
Update zh_cn.json

Removed translate of material.tconstruct.cactus
Removed translate of material.tconstruct.obsidian
Removed translate of material.tconstruct.prismarine
Removed translate of material.tconstruct.netherrack
Removed translate of material.tconstruct.paper
Removed translate of material.tconstruct.sponge
Removed translate of material.tconstruct.firewood
Removed translate of material.tconstruct.slime
Removed translate of material.tconstruct.blueslime
Removed translate of material.tconstruct.blueslime.format
Removed translate of material.tconstruct.magmaslime
Removed translate of material.tconstruct.ardite
Removed translate of material.tconstruct.pigiron
Removed translate of material.tconstruct.string
Removed translate of material.tconstruct.vine
Removed translate of material.tconstruct.slimevine_blue
Removed translate of material.tconstruct.slimevine_orange
Removed translate of material.tconstruct.slimevine_purple
Removed translate of material.tconstruct.blaze
Removed translate of material.tconstruct.reed
Removed translate of material.tconstruct.ice
Removed translate of material.tconstruct.endrod
Removed translate of material.tconstruct.feather
Removed translate of material.tconstruct.leaf
Removed translate of material.tconstruct.slimeleaf_blue
Removed translate of material.tconstruct.slimeleaf_orange
Removed translate of material.tconstruct.slimeleaf_purple
Add translate of block.tconstruct.soul_glass
Add translate of block.tconstruct.soul_glass_pane
Add translate of block.tconstruct.liquid_soul_fluid
Add translate of fluid.tconstruct.liquid_soul
Add translate of fluid.tconstruct.flowing_liquid_soul
Add translate of item.tconstruct.liquid_soul_bucket
Add translate of material.tconstruct.seared_stone
Add translate of material.tconstruct.slimewood
Add translate of material.tconstruct.slimesteel
Add translate of material.tconstruct.nahuatl
Add translate of material.tconstruct.tinkers_bronze
Add translate of material.tconstruct.rose_gold
Add translate of material.tconstruct.pig_iron
Add translate of material.tconstruct.hepatizon
Add translate of material.tconstruct.slime_bronze
Add translate of material.tconstruct.soulsteel

KnightMiner commented:
Alright, looks good so far. You will probably have to stop back in and update the translation later, as there is still more content to add, but this is a good start.

ETStareak commented:
您不会好好翻译就别来瞎搅和，自己用用得了，满地烂私货的垃圾东西也敢上交？欺负作者不懂中文是吧？

TUsama commented:
@KnightMiner Please don't trust a guy who just opens his first PR in GitHub a few days ago. Many modders have received bad translation, the author of Pam's HarvestCraft told me that she once received a translation file, but there was so much inappropriate content in that file, as the word of sex, racial bias, swear. I've heard that a modder gives his Patreon address in his mod's guidebook, but the translator just deletes his address, left his Patreon, which is definitely breaking all the rules.

that is my point: you should just receive the translation PR which is from the translator you can trust(like ET). If the translation PR is from a guy you don't even know, you should just wait for somebody to approve this PR. I don't wanna see this mod suffer from malicious translation.

VeroFess commented:
> 您不会好好翻译就别来瞎搅和，自己用用得了，满地烂私货的垃圾东西也敢上交？欺负作者不懂中文是吧？

1.12的翻译中的"漆黑"系列方块，跟据工艺应当翻译为"烧结"，这是历史错误，并不应该继承

提示部分，我根据了语境进行了适当的意译，我不认为这属于夹带私货

WuzgXY-GitHub commented on src/main/resources/assets/tconstruct/lang/zh_cn.json:
```diff
@@ -0,0 +1,811 @@
+    "item.tconstruct.molten_soulsteel_bucket": "熔融魂钢桶",
+    "block.tconstruct.molten_netherite_fluid": "熔融钙矾石",
+    "fluid.tconstruct.molten_netherite": "熔融钙矾石",
+    "fluid.tconstruct.flowing_molten_netherite": "流动的熔融钙矾石",
```
你是怎么把下界合金翻译成钙矾石的

What on earth make you translate `Netherite` into `Ettringite`

VeroFess replied:
这是他的学术名称，由于匠魂的特殊性

VeroFess replied:
这是他的学术名称，由于匠魂的特殊性我使用了google translate对名词进行了翻译，这是我的失误

TUsama replied:
我从未见过能把自己的无知说的如此轻描淡写的人，但凡你有一点原版游玩经验都不会犯此错误，这是mc自己新造出来的词，你在这装什么？

ETStareak replied:
> 这是他的学术名称，由于匠魂的特殊性我使用了google translate对名词进行了翻译，这是我的失误

钙矾石**（Ettringite）**是一种无色到黄色的钙铝硫酸盐矿物，通常为无色柱状晶体，部分脱水会变白。
“学术名称”，笑死。
机翻光明正大还敢交上来当官译，我谔谔。

ChromaPIE replied:
出示一下你把Netherite判断为“学术名称”引用的文献 以及阐明一下“匠魂的特殊性”是如何影响你把原版确定的术语“下界合金”翻译成“钙矾石”的

VeroFess replied:
这是我因为我的服务器需要所以进行的翻译，本着开源社区的原则，对进行的修改向上游进行反馈，如果您有更好的翻译，请Pull Reuuest而不是来杠我，感谢

ETStareak replied:
> 这是我因为我的服务器需要所以进行的翻译，本着开源社区的原则，对进行的修改向上游进行反馈，如果您有更好的翻译，请Pull Reuuest而不是来杠我，感谢

自知之明很重要，你，没有。这种信息垃圾你敢向上交，你这叫开源精神？你这叫贻笑大方！

VeroFess replied:
因为我对矿物系统并不熟悉，所以我采用了https://translate.google.cn/?sl=en&tl=zh-CN&text=netherite&op=translate所提供的翻译，我并不是专业的翻译人员，如果您有更好的翻译请提交pr,非常感谢

WuzgXY-GitHub commented on src/main/resources/assets/tconstruct/lang/zh_cn.json:
```diff
@@ -0,0 +1,811 @@
+    "gui.tconstruct.fluid.hold_shift": "按住 §e§oShift§r§7 来使用桶",
+    "entity.tconstruct.blue_slime": "蓝色史莱姆",
+    "block.tconstruct.green_slime_dirt": "绿色史莱姆土",
+    "block.tconstruct.green_slime_dirt.tooltip": "block.tconstruct.slime_dirt.tooltip",
```
从这开始的一大摊全是没翻译还把 key 塞进去的

These are all untranslated and filled with translation keys

WuzgXY-GitHub commented on src/main/resources/assets/tconstruct/lang/zh_cn.json:
```diff
@@ -0,0 +1,811 @@
+    "gui.tconstruct.warning": "警告",
+    "gui.tconstruct.general.hover": "将鼠标悬停在条目上以获取更多信息",
+    "gui.tconstruct.error.invalid_pattern": "模板不包含有效的工具零件!",
+    "gui.tconstruct.error.invalid_recipe": "无效的食谱!",
```
还搁这机翻呢，配方翻译成食谱

Machine-translated, `crafting recipe` was translated into `list of cooking ingredients`

VeroFess replied:
此处是由于未能在游戏中找到相应提示采取的翻译

TUsama replied:
可以鉴定完全是小白。

ETStareak replied:
> 此处是由于未能在游戏中找到相应提示采取的翻译

用过英文版的Minecraft能不知道recipe这个单词在游戏中的特定用法？您就一光明正大机翻怪。

ChromaPIE replied:
你作为Minecraft模组的翻译者看到recipe一词会首选“食谱”而不是“配方” 根本说不通的 你是在翻匠魂 不是潘马斯农场披萨工艺懒人厨房

ChirenCQR replied:
> 此处是由于未能在游戏中找到相应提示采取的翻译

但凡你好好去查阅下原版中文wiki都有写到好吧！

WuzgXY-GitHub commented on src/main/resources/assets/tconstruct/lang/zh_cn.json:
```diff
@@ -0,0 +1,811 @@
+    "block.tconstruct.soulsteel_block.tooltip": "可以用作信标的基座",
+    "item.tconstruct.soulsteel_ingot": "魂钢锭",
+    "item.tconstruct.soulsteel_nugget": "魂钢块",
+    "item.tconstruct.netherite_nugget": "钙矾石块",
```
一堆把粒翻译成块的

`Nuggets` were translated into `Blocks`

VeroFess replied:
这是批量替换的错误。非常抱歉

ETStareak replied:
> 这是批量替换的错误。非常抱歉

批量替换，怎么个批量法？批量完了不懂得检查？搞笑不？

WuzgXY-GitHub reviewed (commented):
Except these there are still lots of translation problems, including disunity, non-standard terms, weird word collocations, and even machine-translation

ChromaPIE commented:
> Alright, looks good so far. You will probably have to stop back in and update the translation later, as there is still more content to add, but this is a good start.

Nah it's not. It's a sheer shame to have such a localization work for this marvellous mod. I, and quite a couple of native Chinese speakers, and also TiC lovers, are here to review, and requesting rolling this merged PR back, as it contains high amount of lines tolerating unchecked online traslation results, violating contents to Vanilla terms, and ridiculous mistakes. Should be worthy to just wait for another qualified zh_cn localization.

TUsama commented:
我不知道你是出于什么样的心态将自己的翻译提交到官库的，但我在想问你几个问题：

1. 你确保你的翻译文件的正确性和准确性吗？
2. 你在提交之前有考虑过修改惯用译名的后果吗？
3. 你做好让全国那么多玩家都用你的翻译的准备了吗？

第一是基础，第二积累，第三是责任。你的翻译文件错漏百出，肆意修改惯用音译，提交之后也没有找人帮助你review。所以我认为上述的三点你都没有问过自己，毫无疑问你根本没有与匠魂的流行度所匹配的能力给匠魂提交翻译。开源精神不是你肆意妄为，让别人为你擦屁股的借口。

scottctd commented:
@KnightMiner Hello, I'm a native Chinese speaker and also a modder. I should say that please do **not** use this localization file as your official one because, as people stated above, it is **not qualified**. There are literally **a lot of** stupid mistakes.

VeroFess commented:
这是我因为我的服务器需要所以进行的翻译，本着开源社区的原则，对进行的修改向上游进行反馈，如果您有更好的翻译，请Pull Reuuest而不是来杠我，感谢

我仅仅可以保证在我这里解决了问题，并且这是我能做到的最好的解决方案

再次重申，我仅仅是为了解决"没有中文翻译"的问题，如果您有更好的翻译，请提交PR来解决我的"翻译不准确"的问题

ETStareak commented:
> 这是我因为我的服务器需要所以进行的翻译，本着开源社区的原则，对进行的修改向上游进行反馈，如果您有更好的翻译，请Pull Reuuest而不是来杠我，感谢
> 
> 我仅仅可以保证在我这里解决了问题，并且这是我能做到的最好的解决方案
> 
> 再次重申，我仅仅是为了解决"没有中文翻译"的问题，如果您有更好的翻译，请提交PR来解决我的"翻译不准确"的问题

你什么问题都没解决。粪坑中文要比英文造成的问题更大。删库，滚出去。

ETStareak commented:
> @KnightMiner Hello, I'm a native Chinese speaker and also a modder. I should say that please do **not** use this localization file as your official one because, as people stated above, it is **not qualified**. There are literally **a lot of** stupid mistakes.

这里提一下，不用打扰作者了，今天凌晨我在Discord上聊过这个问题了。他意思是反正现在开发早期，不太当回事。
[我在做了（指新建文件夹），之前也做过匠魂3的汉化准备：pull/4173。(https://github.com/SlimeKnights/TinkersConstruct/pull/4173)
目前在尝试阅读和简单翻译匠魂3的官方谷歌文档。还请大家不要着急，在做了在做了。

VeroFess commented:
这是我所能做到的最好的翻译，如果你想，你当然可以进行翻译，在此之前，我希望你了解开源社区的运作原理，而不是对其他人进行人身攻击。

我不是专业的翻译人员，如果你有这和我对线的闲工夫，麻烦您提交一份精良且优美的翻译，我不胜感激

TUsama commented:
你的观点恰恰证明你自己的问题所在之处：你仅凭你个人的需要，就把你所产出的垃圾翻译提交到官库上来，这与那些 spam PR 在本质上是一样的，你从未考虑过你的翻译会被社区众多玩家采用，你从未考虑过社区玩家的利益，你从来都是因为个人利益提交的翻译，所谓的“开源精神”只是你打的幌子罢了，如果作者能看懂你的 PR 内容，你的 PR 早就被归入了骚扰 PR 之列。骚扰 PR 也配称为“开源精神”？

ETStareak commented:
> > @KnightMiner Hello, I'm a native Chinese speaker and also a modder. I should say that please do **not** use this localization file as your official one because, as people stated above, it is **not qualified**. There are literally **a lot of** stupid mistakes.
> 
> 这里提一下，不用打扰作者了，今天凌晨我在Discord上聊过这个问题了。他意思是反正现在开发早期，不太当回事。
> 我在做了（指新建文件夹），之前也做过匠魂3的汉化准备（[https://github.com/SlimeKnights/TinkersConstruct/pull/4173）
，目前在尝试阅读和简单翻译匠魂3的官方谷歌文档。还请大家不要着急，在做了在做了。

ETStareak commented:
> 这是我所能做到的最好的翻译，如果你想，你当然可以进行翻译，在此之前，我希望你了解开源社区的运作原理，而不是对其他人进行人身攻击。
> 
> 我不是专业的翻译人员，如果你有这和我对线的闲工夫，麻烦您提交一份精良且优美的翻译，我不胜感激

正是因为你搁这儿丢了这么个粪坑，所以我今天一个上午没能去继续做官方文档的翻译，而是在这里尝试擦屁股。
重复一遍，删库，滚出去。

WuzgXY-GitHub commented:
如果你想解决没中文的问题，自己把语言文件塞到服务器的模组文件里不就完事了，犯得着把你那质量奇低的译本交上来吗？我还真是头一次在一个 commit 里见到如此多的问题，长见识了

你自己看看前面的讨论。你这份译本根本不值得被提交，有时候一份差译本不如英文的。

我们没想着杠你，这叫合理指责。你自己交这么一份自知水平不行的翻译还觉得自己做对了，好歹提供了临时替代品，补了空白是吧，我可以告诉你我甚至见过为了补空交机翻的，这么一份机翻，还有你的这个拙劣译本，最终合计起来都不能让玩家得到有了翻译的好处。知不知道 EIO 为什么不接收官中的 PR 啊，还不是因为有这种拿所谓「开源精神」当幌子的根本不考虑整个社区玩家的感受的差翻译存在啊？！

WuzgXY-GitHub commented:
初版翻译极为重要。它如果一直流传下去，最终会因为流传过广不能再被改动。@ETStareak 不顾众人指责，就在这个 TiC 的项目上为修改早期的历史遗留错误汉化作了不少贡献，但有些模组的古早错误译本（比如神秘时代）已经改不动了。如果你也想让自己的拙劣翻译遗臭万年被后世大量玩家指责，你就尽管交，反正我也拦不住。但我一定会尽自己最大的努力阻止作者 merge 你的译本，还请好自为之

VeroFess commented:
您所指出的问题，我正在进行修改，并且会在commit中指出是我的错误，但是请您记住，开源项目仅仅是因为兴趣而推动的，如果您有更加专业，更加健全的翻译，请您积极提出PR对问题进行覆盖，而不是在这里和上一位PR的作者浪费时间

我所能做到的只有我可以做到的最好的，我承认我在翻译方面缺乏相应的能力，所以，请您在任何的开源项目中发现了力所能及可以解决的问题时，先进行PR以修正，感谢

ETStareak commented:
> 您所指出的问题，我正在进行修改，并且会在commit中指出是我的错误，但是请您记住，开源项目仅仅是因为兴趣而推动的，如果您有更加专业，更加健全的翻译，请您积极提出PR对问题进行覆盖，而不是在这里和上一位PR的作者浪费时间
> 
> 我所能做到的只有我可以做到的最好的，我承认我在翻译方面缺乏相应的能力，所以，请您在任何的开源项目中发现了力所能及可以解决的问题时，先进行PR以修正，感谢

又双叒叕重复一遍，删库，滚出去。
我们不可能依照你这版汉化去进行完善和补充，只能也只会依照我们已有的进度进行翻译补充和制作；作者也在Discord上表示，一个开发早期的东西你们急着翻译什么，后面改来改去的麻烦死你。所以我现在主要是对匠魂3的背景资料——长达29页的官方谷歌文档——进行整理和翻译。至于你，早早把别人的测试代码扒下来开服务器，我能说其心可诛吗？
你在这里，只是无意义地侵占本模组和其它模组官方汉化工的工作时间。

VeroFess commented:
我寻思一个MIT协议的模组我自己编译了就其心可诛了？那我建议您不要使用开源的产品。

我进行PR是因为我的兴趣，如果你想，请**自己提交**

LucunJi commented:
> 我寻思一个MIT协议的模组我自己编译了就其心可诛了？那我建议您不要使用开源的产品。

那么我这里引用一下《自由软件，自由社会》的一段文字：

> Thus, therhetoric of “open source” focuses on the potential to make high-quality, powerfulsoftware, but shuns the ideas of freedom, community, and principle.
> Richard Stallman, _Free Software, Free Society: Selected Essays of Richard Stallman_. Free Software Foundation.

开源不等于低质量。相反，开源的目的之一就是为了保证产品的质量和安全。因此，对低质量的 pr 进行批判也是开源的一环。
**不爽就不要提到开源。**

Snapshotlight commented:
不会真有人觉得“凭自己的兴趣”就可以随随便便给一个模组提交翻译pr吧？
麻烦您在翻译的时候想一想这是给谁的。
把一个垃圾产品传递给所有这个模组的使用者才是真正的不负责任

VeroFess commented:
1. 这是**我所能做到的最好的，你比我做的好是你厉害**
2. 我也没不让你批判啊
3. 但是毫无逻辑的硬杠就不合适了吧，我自己编译个放到服务器上咋滴你了？

Snapshotlight commented:
> 1. 这是**我所能做到的最好的，你比我做的好是你厉害**
> 2. 我也没不让你批判啊
> 3. 但是毫无逻辑的硬杠就不合适了吧，我自己编译个放到服务器上咋滴你了？

我再强调一遍，您的翻译是给所有人看的，至于”关你p事“，把垃圾翻译提交给所有使用者看就跟我有关系了，我也要玩这个模组，我可忍受不了您的翻译。
对了补充一点，不要随便给人扣帽子，我想我的话并不是没有逻辑的。

VeroFess commented:
这里有一篇答疑[GitHub 的 Pull Request 是指什么意思？](https://www.zhihu.com/question/21682976) ，希望各位可以去看看

Snapshotlight commented:
> 这里有一篇答疑[GitHub 的 Pull Request 是指什么意思？](https://www.zhihu.com/question/21682976) ，希望各位可以去看看

如果您真的想以此辩护，我想说，您的pr是一次不合格的改卷

ETStareak commented:
> 这里有一篇答疑[GitHub 的 Pull Request 是指什么意思？](https://www.zhihu.com/question/21682976) ，希望各位可以去看看

问题来了，作者不知道你的修改是不是正确，没人管他只能先用着。这仍然不符合开源精神。
“欺负作者不懂中文是吧？”

VeroFess commented:
最后再强调一次，这是**我**认为**我**将问题解决了的方案，并且**我**希望这个**我认为**的解决方案分享给其他人，**你**翻译的好请**你**告诉大家什么是**更加正确**的解决方案，**术业有专攻**，**这是我的水平所能做到的最好的翻译**，**并且我在翻译完成之后没有觉得它有啥问题，所以就PR了，如果我的水平作者认为是不合适的，他就不会将其Merge**

ETStareak commented:
> 最后再强调一次，这是**我**认为**我**将问题解决了的方案，并且**我**希望这个**我认为**的解决方案分享给其他人，**你**翻译的好请**你**告诉大家什么是**更加正确**的解决方案，**术业有专攻**，**这是我的水平所能做到的最好的翻译**，**并且我在翻译完成之后没有觉得它有啥问题，所以就PR了，如果我的水平作者认为是不合适的，他就不会将其Merge**

作者今天凌晨（东八区）的时候跟我说，他根本不知道翻译好不好，只是你这个PR时间比较近他就Merge了，他根本不知道你这是粪坑。你还有什么好说的？

VeroFess commented:
没人管他只能先用着

> > 这里有一篇答疑[GitHub 的 Pull Request 是指什么意思？](https://www.zhihu.com/question/21682976) ，希望各位可以去看看
> 
> 问题来了，作者不知道你的修改是不是正确，没人管他只能先用着。这仍然不符合开源精神。
> “欺负作者不懂中文是吧？”

这上面至少有五个人在这里回复，如果您对我的翻译有意见，可以向作者PR，或者我也已经跟据各位的建议修改并且提交的新的PR(https://github.com/SlimeKnights/TinkersConstruct/pull/4259)

**有啥问题就直接提，我改，你觉得我改不好就直接PR，在这里BB个啥玩意？**

ChirenCQR commented:
> 最后再强调一次，这是**我**认为**我**将问题解决了的方案，并且**我**希望这个**我认为**的解决方案分享给其他人，**你**翻译的好请**你**告诉大家什么是**更加正确**的解决方案，**术业有专攻**，**这是我的水平所能做到的最好的翻译**，**并且我在翻译完成之后没有觉得它有啥问题，所以就PR了，如果我的水平作者认为是不合适的，他就不会将其Merge**

得了吧，作者是不会中文才会merge的，既然你自己都清楚你自己的翻译水平，你的翻译就只在你的服务器里用用得了，还提交上来干嘛？星落（ET）都说了新的翻译他正在制作，这模组还处于测试版，你着什么急？？

ChitoseRaame commented:
> 最后再强调一次，这是**我**认为**我**将问题解决了的方案，并且**我**希望这个**我认为**的解决方案分享给其他人，**你**翻译的好请**你**告诉大家什么是**更加正确**的解决方案，**术业有专攻**，**这是我的水平所能做到的最好的翻译**，**并且我在翻译完成之后没有觉得它有啥问题，所以就PR了，如果我的水平作者认为是不合适的，他就不会将其Merge**

如果作者知道你翻译的是什么**,那么他根本不会接受你这个pull请求.
什么叫本地化?我不知道这门语言,我把这个工作交出去,我不会我怎么能够审核呢?

WuzgXY-GitHub commented:
> 最后再强调一次，这是**我**认为**我**将问题解决了的方案，并且**我**希望这个**我认为**的解决方案分享给其他人，**你**翻译的好请**你**告诉大家什么是**更加正确**的解决方案，**术业有专攻**，**这是我的水平所能做到的最好的翻译**，**并且我在翻译完成之后没有觉得它有啥问题，所以就PR了，如果我的水平作者认为是不合适的，他就不会将其Merge**

你搁这扯啥玩意呢，作者有没有能力辨别你的翻译之好与坏你自己心里没点数吗，人家一群不懂中文的开发者出于对游戏的热爱和认为大家都有因这层热爱而有的不会乱交差翻译的责任心把你的 PR 合了你还觉得人家看懂你的翻译并已给出好的评价了是吗

另一层，你觉得自己的 PR 质量没问题才将之上交，那我确实没办法，这好歹你没交自己明知质量不行的汉化，算是到位了。可是我们在这说这么半天不是没原因的，现在我们说了这么多你总该看见质量问题了吧？你还认为这是值得被合的汉化吗？？

VeroFess commented:
你们会翻译，**请自己提交PR**，你们在这里说了半天我已经把你们指出的问题提交了**新的PR**，要是**新的PR**还有问题麻烦**直接指出来**，请不要**扯一些有的没的**

TUsama commented:
刚好相反，人家看得懂就绝对不会 merge 你的，你就是来这里倒垃圾的，什么时候你这也能够称为解决方案了？我们还没到吃屎解决问题的时候。
你总强调“我认为”“我认为”，但事实是：恰恰你所认为的就是错的，是有问题的，从最开始就是如此。因为你的傲慢，你不能理解别人的观点，始终觉得自己拉出来的屎是块宝，还想着分享到开源社区；遇到专有名词也不懂得充分利用搜索引擎，最后产出了极为离谱的翻译，这些都是赤裸裸的证据。
你直到现在都还没能理解我们批评你的点在哪里：你的翻译差是**一部分**，但是**罪不致死**；**但你想要把你的屎强行分享给所有的玩家，那就是与我们相对立，与所有玩匠魂的玩家对立，这才是死罪。**

ETStareak commented:
> 你们会翻译，**请自己提交PR**，你们在这里说了半天我已经把你们指出的问题提交了**新的PR**，要是**新的PR**还有问题麻烦**直接指出来**，请不要**扯一些有的没的**

你的意思是说，因为你要用自己构建的测试版源码开服，所以我就得在模组Alpha还遥遥无期的时候为你把汉化准备好，对吧？

WuzgXY-GitHub commented:
> 你们会翻译，**请自己提交PR**，你们在这里说了半天我已经把你们指出的问题提交了**新的PR**，要是**新的PR**还有问题麻烦**直接指出来**，请不要**扯一些有的没的**

我不提这些上位想法了，我只想说你的翻译质量不行并且你都不自知还觉得挺好，希望你**下次别再给公共项目交汉化了**，没有的话自己做完自己用得了。

考虑到作者的邮箱可能已经完蛋了，还是尽量少说吧

VeroFess commented:
首先，我没有主分支任何的**写入权限**，其次，我在网上**没有找到任何可用的1.16翻译**，再次，在我之前已经有一份**基于1.12的翻译作者没有合并，我认为作者是有判断的水平的**，最后，**因为1.12将烧结翻译成漆黑，所以我优先信任谷歌翻译给出的结果而不是之前版本的翻译**

ETStareak commented:
> 首先，我没有主分支任何的**写入权限**，其次，我在网上**没有找到任何可用的1.16翻译**，再次，在我之前已经有一份**基于1.12的翻译作者没有合并，我认为作者是有判断的水平的**，最后，**因为1.12将烧结翻译成漆黑，所以我优先信任谷歌翻译给出的结果而不是之前版本的翻译**
我查了查宁的活动记录，没看见这么个PR。给链接。

ChirenCQR commented:
> 首先，我没有主分支任何的**写入权限**，其次，我在网上**没有找到任何可用的1.16翻译**，再次，在我之前已经有一份**基于1.12的翻译作者没有合并，我认为作者是有判断的水平的**，最后，**因为1.12将烧结翻译成漆黑，所以我优先信任谷歌翻译给出的结果而不是之前版本的翻译**

你是不是听不懂上面人讲的话？作者要是有判断中文翻译是否正确的水平的话，他不会自己写中文翻译吗？还轮到你在这乱叫？

VeroFess commented:
PR的意义在于**个人将自己认为的解决方案分享给其他人**, 我不认为我是万能的，我也没认为其他人是万能的，这个PR在正式被Merge前有整整**三天时间**，各位这么关心汉化质量请不要在这里**做事后诸葛亮**

VeroFess commented:
> > 首先，我没有主分支任何的**写入权限**，其次，我在网上**没有找到任何可用的1.16翻译**，再次，在我之前已经有一份**基于1.12的翻译作者没有合并，我认为作者是有判断的水平的**，最后，**因为1.12将烧结翻译成漆黑，所以我优先信任谷歌翻译给出的结果而不是之前版本的翻译**
> > 我查了查宁的活动记录，没看见这么个PR。给链接。

https://github.com/SlimeKnights/TinkersConstruct/pull/4232

GreyElaina commented:
您.

> 你们会翻译，请自己提交PR

谢谢, 但是为什么要调转矛头呢?

> 我在网上没有找到任何可用的1.16翻译

@1.12.2, @1.7.10, 您不会参考下?

> PR的意义在于个人将自己认为的解决方案分享给其他人

赶紧 fork 吧.

> 做事后诸葛亮

还没 merge 呢.

GreyElaina commented:
哦, 已经 merged 了, 没事了

VeroFess commented:
> 您.
> 
> > 你们会翻译，请自己提交PR
> 
> 谢谢, 但是为什么要调转矛头呢?
> 
> > 我在网上没有找到任何可用的1.16翻译
> 
> @1.12.2, @1.7.10, 您不会参考下?
> 
> > PR的意义在于个人将自己认为的解决方案分享给其他人
> 
> 赶紧 fork 吧.
> 
> > 做事后诸葛亮
> 
> 还没 merge 呢.

因为1.12将烧结翻译成漆黑，所以我优先信任谷歌翻译给出的结果而不是之前版本的翻译

TUsama commented:
你所说的根本不能为你向官库提交翻译这一行为做辩护，你只是在陈述自己如何理所当然的挑选出了一份满意的翻译而已。你自己都说了“没有合并”，你难道不会考虑为什么没被合并吗？说白了，你的这一行为是对公共领域的入侵、破坏，这是因为你做这件事情完全没有考虑清楚，把公共领域当作自己的后花园肆意破坏，我们必须制止你的这一行为。

GreyElaina commented:
哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈

> 优先信任谷歌翻译给出的结果而不是之前版本的翻译

你就是词典大师? Google 在自造词这块您心里没点 Number 么?

ETStareak commented:
> 您.
> 
> > 你们会翻译，请自己提交PR
> 
> 谢谢, 但是为什么要调转矛头呢?
> 
> > 我在网上没有找到任何可用的1.16翻译
> 
> @1.12.2, @1.7.10, 您不会参考下?
> 
> > PR的意义在于个人将自己认为的解决方案分享给其他人
> 
> 赶紧 fork 吧.
> 
> > 做事后诸葛亮
> 
> 还没 merge 呢.

这里说一下，不能参考，不能参考，不能参考。匠魂1和2区别很大，结果匠魂2有很多错误的照搬（在PR#4173里修复了）。匠魂3最好是先解决官方文档（https://docs.google.com/document/d/1L0zG8BOXVXaCLpnZFpgwS2KZjPpyAZSZvzcHpvNzOHw/），我在做了。

mamaruo commented:
原版术语 QA 很重要，有一个 CAT 软件或许能让您的翻译之路走得更顺。

GreyElaina commented:
> 匠魂2有很多错误的照搬

我也没让你照搬啊.

ETStareak commented:
> > 匠魂2有很多错误的照搬
> 
> 我也没让你照搬啊.

哦草，我理解有误，没事了。

EnterFor commented:
别打了，你们不要再打了...

VeroFess commented:
> 你所说的根本不能为你向官库提交翻译这一行为做辩护，你只是在陈述自己如何理所当然的挑选出了一份满意的翻译而已。你自己都说了“没有合并”，你难道不会考虑为什么没被合并吗？说白了，你的这一行为是对公共领域的入侵、破坏，这是因为你做这件事情完全没有考虑清楚，把公共领域当作自己的后花园肆意破坏，我们必须制止你的这一行为。

您可以制止啊，我求求您**PULL REQUEST**，你在我一没写入权限的人这里BB啥呢？再说一遍我**没有写入权限**，我也**不负责审核**，我也没能力保证我是**一定正确的**，在被合并之前有整整**三天**的时间，**在合并之后解决问题的方法是开ISSUSE或者编写新的PR而不是在这里杠**

GreyElaina commented:
![image](https://user-images.githubusercontent.com/31543961/106374255-448c9f00-63bc-11eb-9414-50c14d6d51bb.png)

这就是优先考虑词典来源, 将 Google Translation 当为学术参考, 我只能谔谔,,,,

VeroFess commented:
新的PR在https://github.com/SlimeKnights/TinkersConstruct/pull/4259，有问题就挑，我改，你自己PR也行，在这里说啥呢？有这时候诸葛亮的劲那您去帮我挑挑错可以不

GreyElaina commented:
作者本人是不懂 Chinese 的, 换作我, 如果有人给我项目提交了 German 的 Translation, 我也没法评判他翻译质量...

GreyElaina commented:
好了, 我们可以去 review 您的新 PR 了.

VeroFess commented:
> 作者本人是不懂 Chinese 的, 换作我, 如果有人给我项目提交了 German 的 Translation, 我也没法评判他翻译质量...

所以这不是有社区么，没社区我也不知道我翻译错了啊，我也不知道我翻译的烂啊，我也不知道作者就这么随便放我PR过了啊，指出问题，我改了，还有啥问题么，我也没死磕着"我的翻译全天下最牛逼啊"

TUsama commented:
你自己看看：足足三天，你没找一个人帮助你reviewe，这不就是傲慢吗？这不就是觉得自己的工作没问题吗？你反复强调自己不保证一定正确，但是又揣着傲慢和矜持，没有找人来协助。你自己都把你自己的傲慢说得一清二楚了。

直到现在你还没明白：我们当然可以再次开一个PR，但是你破坏公共领域的罪如何清算？你所犯下的罪不是改正了仓库内容就能赎清的，你必须接受来自社区的怒火，因为你漠视了它们，现在就是你赎罪的时候，好好担着吧。

ETStareak commented:
> > 作者本人是不懂 Chinese 的, 换作我, 如果有人给我项目提交了 German 的 Translation, 我也没法评判他翻译质量...
> 
> 所以这不是有社区么，没社区我也不知道我翻译错了啊，我也不知道我翻译的烂啊，我也不知道作者就这么随便放我PR过了啊，指出问题，我改了，还有啥问题么，我也没死磕着"我的翻译全天下最牛逼啊"

行了行了，您别折腾了，自己想用自己用去，我们管不着。别祸祸社区就行，关了您那新PR吧。

WuzgXY-GitHub commented:
没找我们 review 却说我们是事后诸葛亮，难道我们每天 24 小时盯着整个 GitHub 的库 review 吗

VeroFess commented:
> 你自己看看：足足三天，你没找一个人帮助你reviewe，这不就是傲慢吗？这不就是觉得自己的工作没问题吗？你反复强调自己不保证一定正确，但是又揣着傲慢和矜持，没有找人来协助。你自己都把你自己的傲慢说得一清二楚了。
> 
> 直到现在你还没明白：我们当然可以再次开一个PR，但是你破坏公共领域的罪如何清算？你所犯下的罪不是改正了仓库内容就能赎清的，你必须接受来自社区的怒火，因为你漠视了它们，现在就是你赎罪的时候，好好担着吧。

首先，有write权限的人**必须负责对PR进行审核**，其次**开源社区的每一个人都应该进行reviewe**

VeroFess commented:
> 你一开始就没想着找社区吧……扯什么事后诸葛亮，没找我们 review，难道我们每天 24 小时盯着整个 GitHub 的库 review 吗

我和您很熟么？您哪位，我咋知道您可以review?

WuzgXY-GitHub commented:
> 
> 
> > 你一开始就没想着找社区吧……扯什么事后诸葛亮，没找我们 review，难道我们每天 24 小时盯着整个 GitHub 的库 review 吗
> 
> 我和您很熟么？您哪位，我咋知道您可以review?

抱歉，我现在修改原句用语

TUsama commented:
你看看，要求别人做到自己不可能做到的事，这不就是**傲慢**吗？要求不懂中文的作者进行中文翻译的审查，这不就是**傲慢**吗？要求社区成员自动知悉你名不见经传的 PR 并帮助你 review ，这不就是**傲慢**吗？你到现在还没有想清楚发生了什么？**是你先抛弃了社区，社区才会抛弃你**！只要你拉的下脸来，社区会不帮助你吗？说到底不过仍是**“你以为”**罢了！

VeroFess commented:
这右上角不是有个"WATCH"么？我也没听说过PR需要啥人的同意啊？你们这些匠魂关注者都不WATCH他究竟发生什么新变化么？我一个业余的人还点了WATCH就为了收到作者更新了啥，这就是你们说的“关注”？我寻思我PR的名称清楚的写着“提交了1.16的翻译”，这在邮件里咋样都能看到我在干嘛吧？三天不看一眼邮箱事后来这里叭叭叭真就社区支柱呗？

KnightMiner commented:
Everyone on this tracker, please relax. The mod has not been released yet. As you may have guessed, I do not speak chinese, I have no way to know whether a translation is good or bad. If someone submits a PR, the best I can do is check to make sure its not spam using translation software, I cannot easily verify whether they speak the language.

If this language file is as bad as you guys say, I can remove it. But arguing about it in the comments of an english issue tracker in a language I do not understand does not help me do that. In any case, as I have said, its not released yet, so this has not done as much harm as the spam of comments would lead someone to believe.

----

用Google翻译翻译

追踪器上的所有人，请放松。 该mod尚未发布。 您可能已经猜到了，我不会说中文，也无法知道翻译的好坏。 如果有人提交了PR，我能做的最好的就是检查一下，以确保它不使用翻译软件发送垃圾邮件，我无法轻易地验证他们是否说这种语言。

如果此语言文件与您说的一样糟糕，我可以将其删除。 但是在英语问题跟踪器的注释中以我不懂的一种语言来争论它并不能帮助我做到这一点。 无论如何，正如我已经说过的那样，它尚未发布，因此并没有像评论垃圾邮件那样令人信服。

VeroFess commented:
I am very frustrated with this matter. If my translation is really as bad as you said, then please at least tell me what errors remain after the repair. But here only one friend really pointed out the problem, other than that is meaningless quarrel. The meaning of open source is that everyone can try to make things they like better, and I just try my best to do what I can do.

KnightMiner commented:
At this time, I reverted the existing translation until this can be sorted out. It does not benefit us to have multiple competing translators. I suggest you find a way to collaborate with the others seeking to translate into Chinese to prevent wasted work and to end up with the highest quality translation. 

I don't want anyone to be deterred from contributing, but I also have no way to verify statements from either group seeking to translate.

----

用Google翻译翻译

这时，我还原了现有翻译，直到可以解决为止。 拥有多个竞争翻译器对我们没有好处。 我建议您找到一种与其他寻求翻译成中文的人合作的方法，以防止浪费工作并最终获得最高质量的翻译。

我不想阻止任何人做出贡献，但是我也无法验证任何试图翻译的组织的陈述。

VeroFess commented:
> At this time, I reverted the existing translation until this can be sorted out. It does not benefit us to have multiple competing translators. I suggest you find a way to collaborate with the others seeking to translate into Chinese to prevent wasted work and to end up with the highest quality translation.
> 
> I don't want anyone to be deterred from contributing, but I also have no way to verify statements from either group seeking to translate.
> 
> 用Google翻译翻译
> 
> 这时，我还原了现有翻译，直到可以解决为止。 拥有多个竞争翻译器对我们没有好处。 我建议您找到一种与其他寻求翻译成中文的人合作的方法，以防止浪费工作并最终获得最高质量的翻译。
> 
> 我不想阻止任何人做出贡献，但是我也无法验证任何试图翻译的组织的陈述。

I am seeking help from a commercial translation agency to improve the quality of my translation. I cannot guarantee the quality of my translation at this time, so sorry.

3TUSK commented:
If anyone needs help on proofreading, I am available for proofreading requests. Feel free to ping me in the PR in question.
